test/e2e/storage/persistent_volumes-local.go

```diff
@@ -118,8 +118,8 @@ const (
 	// volumeConfigName is the configmap passed to bootstrapper and provisioner
 	volumeConfigName = "local-volume-config"
 	// bootstrapper and provisioner images used for e2e tests
-	bootstrapperImageName = "quay.io/external_storage/local-volume-provisioner-bootstrap:v1.0.1"
-	provisionerImageName  = "quay.io/external_storage/local-volume-provisioner:v1.0.1"
+	bootstrapperImageName = "quay.io/external_storage/local-volume-provisioner-bootstrap:latest"
```

Comment:
I just pushed v2.0.0 for both of these images

Reply:
Updated.